docs/journalist/faq.rst

```diff
@@ -139,16 +139,17 @@ without opening it in an interactive viewer application.
 
 Why can't I copy and paste?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
-In the current version, copy and paste between and to SecureDrop Workstation VMs
-is disabled for security reasons. The goal of this restriction is to minimize
-the risk of accidental pastes of sensitive content, and to reduce the attack
-surface for attempts to exfiltrate information.
+You should be able to copy and paste *within* any VM on the system, e.g.,
+from one application running in ``sd-app`` to another.
```

Comment:
this is a nit, but I wonder if we should pick a different example vm to discourage running any other app than the SecureDrop client in `sd-app`?

Reply:
Ah, and just a wee merge conflict due to #36 being merged.

Reply:
I'm fine with changing this example to copying text within `work` (a default Qubes VM) to sidestep the question. I do think we should talk a bit more about such use cases as they arise -- e.g., the boilerplate use case. I'm not sure it really _is_ better to recommend using a separate VM in a case like that, because then the user has to punch in RPC holes to allow cross-VM copy/paste, and manage the copy/paste opsec every time -- plus the VM may well have worse security properties than `sd-app`, if we don't ship it. But this is probably not something we want to speak to in a casual example.

Reply:
Changed to a different example in b07ac1948fe4c804320087525e85901fb183f269